subgraphs/notional-finance/src/common/constants.ts

```diff
@@ -0,0 +1,184 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+///////////////////
+///// Versions /////
+////////////////////
+
+export const PROTOCOL_SCHEMA_VERSION = "2.0.1";
```

Comment:
Can be removed. We are using this auto generated file that grabs the versions from the `deployment.json` now. You can see it being used here: https://github.com/messari/subgraphs/blob/master/subgraphs/aave-v2-forks/src/helpers.ts#L94

Note; we place it outside the getter to update if we have to graft